ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/HomeScreenMapContainerViewModel.kt

```diff
@@ -122,8 +124,10 @@ internal constructor(
         .toLiveData()
 
     suggestLoiJobs =
-      surveyRepository.activeSurveyFlow.map {
-        it?.jobs?.filter { job -> job.suggestLoiTaskType != null }?.toList() ?: listOf()
+      surveyRepository.activeSurveyFlow.combine(cameraZoomUpdates.asFlow()) { survey, zoomLevel ->
```

Comment:
Can this be rewritten using Coroutines/Flow?

Reply:
This is using Flow

Reply:
suggestLoiJobs is a Flow<List<Job>>

Reply:
Oops sorry I missed that!